src/lib.rs

```diff
@@ -305,3 +337,42 @@ pub enum ClientCapability {
     /// The driver provides atomic modesetting
     Atomic = drm_ffi::DRM_CLIENT_CAP_ATOMIC as u64,
 }
+
+/// Used to specify a vblank frame to wait for
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum VblankWaitFrame {
+    /// Wait for a specific vblank sequence number
+    Absolute(u32),
+    /// Wait for a given number of vblanks
+    Relative(u32),
+}
```

Comment:
If you think this is a nice API, it could perhaps also be employed in `fn page_flip` in the `control` module, which currently uses a `(PageFlipTarget, u32)` type for this (alternatively use that format with bitflag enum here?).